source/edge-server.txt

```diff
@@ -0,0 +1,126 @@
+
+   You have a warehouse, retail store, air-gapped laboratory, or another 
+   premises where devices should be able to seamlessly sync their data.
+   Those locations may or may not be online at any given time. Lack of a 
+   network connection shouldn't prevent the devices from sharing data
+   with each other.
+
+   Edge Server runs on-premises and handles sync between local devices,
+   and bi-directional sync between the Edge Server and Atlas.
+
+   Edge Server is currently in private preview. If you are interested in 
+   previewing the Atlas Device Sync Edge Server, complete this form.
+
+   .. button:: I'm interested!
+      :uri: https://forms.gle/3TgGK9cUGenTAQYH7
+
+.. image:: /images/plp/Spot_AzureBlue_Sync_Tech_RealmSync.png
+   :alt: Atlas Device Sync Illustration
+
+
+.. kicker:: What It Is
+
+Key Features
+------------
+
+.. card-group::
+   :columns: 3
+   :style: compact
+
+   .. card::
+      :headline: MongoDB + Sync - On Premises
+      :icon: /images/icons/General_FEATURES_IntelligentEdge3x.png
+      :icon-alt: An icon depicting a mobile phone connecting with the cloud
+
+      Edge server is a local instance of MongoDB paired with a sync server 
+      that runs on-premises. It brings real-time sync, disconnection 
+      tolerance, and conflict resolution to the location's devices.
+
+   .. card::
+      :headline: Optimize Bandwidth
+      :icon: /images/icons/Technical_REALM_ConflictResolution3x.png
+      :icon-alt: An icon depicting conflict resolution
+
+      Delta sync ensures that only fields with changes in a document are 
+      sent to Atlas. Mission-critical data is transferred efficiently while 
+      minimizing network congestion.
+
+   .. card::
+      :headline: Client Independence
+      :icon: /images/icons/Technical_REALM_Offline3x.png
+      :icon-alt: An icon depicting network loss
+
+      Edge clients can sync and access essential data independently, even 
+      when disconnected from the central database. Applications and devices 
+      in a connected infrastructure continue to function seamlessly, 
+      even with intermittent connectivity.
+
+.. kicker:: How it Works
+
+Edge Server Setup
+-----------------
+
+.. procedure::
+
+   .. step:: Create a Device Sync-Enabled App
+
+      Create an :ref:`App Services App <create-app>` with 
+      :ref:`Device Sync enabled <enable-sync>`.
+
+      To get started quickly, create a template app.
+
+      Once your App is created, coordinate with your Product or Account 
+      Representative to enable Edge Server.
+
+      .. button:: Start Building with a Template App
+         :uri: https://www.mongodb.com/realm/register?tck=docs_CTA_app_services
+
+   .. step:: Configure the Edge Server Host
+
+      Configure hardware on-site or a cloud server to host your Edge Server.
+      Refer to the documentation for required network access, hardware 
+      requirements, and supported operating systems.
+
+   .. step:: Install and Configure the Edge Server
```

Comment:
did we want to provide the link for downloading the edge server here or no?

Reply:
The link will be available on the "Configure" page, and I'm reluctant to provide too many details here as this is intended to be more of a high-level overview to quickly establish what the Edge Server is and broadly how to use it. I'll be adding links to the pages in the section once the section is built out, though.